domenetjenester/uttak/src/main/java/no/nav/foreldrepenger/domene/uttak/uttaksgrunnlag/fp/FastsettUttaksgrunnlagTjeneste.java

```diff
@@ -108,6 +107,49 @@ private OppgittFordelingEntitet justerFordeling(UttakInput input, LocalDate endr
         return new OppgittFordelingEntitet(kopier(justertePerioder), fordeling.getErAnnenForelderInformert(), fordeling.ønskerJustertVedFødsel());
     }
 
+    private static boolean skalJustereFordelingEtterFamiliehendelse(ForeldrepengerGrunnlag fpGrunnlag, List<OppgittPeriodeEntitet> perioder, Long behandlingId) {
+        if (behandlingId.equals(3066799L)) { // TODO: Fjern denne etter behandling OK.
+            LOG.info("Justere ikke behandling {}", behandlingId);
+            return false;
+        }
+
+        if (!fpGrunnlag.getFamilieHendelser().gjelderTerminFødsel()) {
+            return false;
+        }
+
+        if (perioder.isEmpty()) {
+            LOG.info("Skal ikke fødselsjustere når gjeldende behandling ikke har uttak (f.eks. ved opphør, berørt, osv)");
+            return false;
+        }
+
+        if (fpGrunnlag.getOriginalBehandling().isPresent()) {
+            if (nySøknadPåTermin(fpGrunnlag.getFamilieHendelser())) {
```

Comment:
Sjekket var lenger inn i koden før som førte til at alle disse periodene gjennomgikk følgende justering

```
var justert = sorterEtterFom(oppgittePerioder);
return slåSammenLikePerioder(justert);

```
Noe som det ikke gjøres nå. Okey?

Reply:
Kanskje flytte denne så vi slår sammen uansett om fødselsjusteringen kalles eller ikke

Reply:
Tenker du da ut til FastsettUttaksgrunnlagTjeneste.justerFordeling()?

Reply:
```
        justertePerioder = sorterEtterFom(justertePerioder);
        justertePerioder = slåSammenLikePerioder(justertePerioder);
        ForeldrepengerGrunnlag fpGrunnlag = input.getYtelsespesifiktGrunnlag();
        if (skalJustereFordelingEtterFamiliehendelse(fpGrunnlag, justertePerioder, behandlingId)) {
            justertePerioder = justerFordelingEtterFamilieHendelse(fpGrunnlag, justertePerioder, ref.relasjonRolle(), fordeling.ønskerJustertVedFødsel());
            LOG.info("Justerte perioder etter flytting ved endring i familiehendelse {}", justertePerioder);
        }
```
Slik du ser for deg da?